### DISCUSSION TITLE
S-HOOK 좋아요 정합성 이슈

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#79] [General]
# S-HOOK 좋아요 정합성 이슈

⚖️ 공부한 것, 일기를 적는 개인 위키입니다 인덱스 페이지: [ wiki-index ] 이번 달의 TIL: [ 2023-08|2023년 8월의 TIL✨ ] 현재 자주 쓰는 글들: [ wooteco|우아한테크코스 ], [ shook|…

https://vero.wiki/shook-like-issue/

**Comment by carsago** [comment]
이거 좀 재밌는 이슈네요

**Reply by Cyma-s** [reply]
여기가 좀 더 자세해요 https://vero.wiki/shook-like-concurrency-issue/ 
근데 확실히 해결하기 어려울 수도... 정책을 바꿀 수도 있겠어요
